app/layout.tsx

```diff
@@ -21,6 +26,19 @@ const RootLayout: FC<Children> = ({ children }: Readonly<Children>) => {
     <html lang="en">
       <body className={inter.className}>
         <AppRouterCacheProvider>{children}</AppRouterCacheProvider>
+        <AppBar>
+          <Toolbar>
+            <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+              <Link href="/">ChessTeacher</Link>
+            </Typography>
+            <Button color="inherit">
+              <Link href={"/chess"}>Chess</Link>
+            </Button>
+            <Button color="inherit">
+              <Link href={"/account"}>Account</Link>
+            </Button>
+          </Toolbar>
+        </AppBar>
```

Comment:
You cannot just place this here. Have you looked at line 28 and what `AppRouterCacheProvider` is?

Reply:
Agree. Has now been placed in page.tsx homepage for the time being until i know where to put it